src/braket/circuits/circuit.py

```diff
@@ -1063,7 +1064,7 @@ def to_ir(self) -> Program:
         )
 
     def as_unitary(self) -> np.ndarray:
-        """
+        r"""
         Returns the unitary matrix representation of the entire circuit.
```

Comment:
Could we update to say `in little endian` format here ?

Reply:
Will do!